crowdsourcing/models.py

```diff
@@ -559,3 +559,36 @@ class TaskComment(models.Model):
     task = models.ForeignKey(Task, related_name='taskcomment_task')
     comment = models.ForeignKey(Comment, related_name='taskcomment_comment')
     deleted = models.BooleanField(default=False)
+
+
+class FinancialAccount(models.Model):
+    owner = models.ForeignKey(UserProfile, related_name='account_owner', null=True)
```

Comment:
it's suggested for many to many relationships else it's main purpose is to have unique constraint names, but surely this related_name was not chosen wisely :D